### PR TITLE
Doc/wording

### PR DESCRIPTION
**Pull request by seanpearsonuk**

### CODE REVIEW

**Comment by seanpearsonuk** [comment]
#1445

File: doc/source/api/general/index.rst [hunk 1]
```diff
@@ -6,73 +6,72 @@ General
...
+:ref:`ref_data_transfer` describes how to transfer mesh data between PyFluent sessions.
 
 Journaling
 ##########
 
-:ref:`ref_journaling` consists of read-write of journal.
-
-Meta
-####
+:ref:`ref_journaling` explains how to read an write Python journals that are reusable between PyFluent and Fluent.
```
Comment (dnwillia-work) [inline doc/source/api/general/index.rst:44]:
```suggestion
:ref:`ref_journaling` explains how to read and write Python journals that are reusable between PyFluent and Fluent.
```
```suggestion
:ref:`ref_journaling` explains how to read an write Python journals that are reusable between PyFluent and Fluent.
```

File: src/ansys/fluent/core/launcher/launcher.py [hunk 2]
```diff
@@ -506,52 +510,51 @@ def launch_fluent(
...
     mode : str, optional
         Launch mode of Fluent to point to a specific session type.
         The default value is ``None``. Options are ``"meshing"``,
         ``"pure-meshing"`` and ``"solver"``.
     server_info_filepath: str
-        Path to server-info file written out by Fluent server. The default is ``None``.
+        Path to server-info file written out by Fluent server. The default is
+        ``None``. ``server_info_filepath`` can be specified either if ``start_instance``
+        is ``True``, where PyFluent will launch and then direct Fluent to write the file
+        at that path, or if ``start_instance`` is ``False``, where PyFluent will use the
```
Comment (mkundu1) [inline src/ansys/fluent/core/launcher/launcher.py:541]: > then direct Fluent to write the file at that path

@seanpearsonuk I don't think this is supported

File: src/ansys/fluent/core/launcher/launcher.py [hunk 3]
```diff
@@ -506,52 +510,51 @@ def launch_fluent(
...
+        If provided, the case file at ``case_filepath`` is read into the Fluent session.
     case_data_filepath : str, optional
-        If provided, reads a fluent case and data file and sets the required settings
-        in the fluent session
+        If provided, the case and data files at ``case_data_filepath`` are read into the Fluent session.
     lightweight_mode: bool, optional
-        Whether to use light io mode for reading case via ``case_filepath`` parameter.
-        This parameter is used only when ``case_filepath`` is provided. The default is
-        ``False``.
+        Whether to read only the lightweight settings from any specified Fluent case file, ignoring the mesh.
```
Comment (mkundu1) [inline src/ansys/fluent/core/launcher/launcher.py:531]: The lightwight_mode reads the lightweight settings in the current solver session. The mesh will be read in a background solver session which will replace the current solver session after applying any change in the lightweight settings made by user in the current session.

File: src/ansys/fluent/core/launcher/launcher.py (hunk 2 above)
Comment (seanpearsonuk) [reply src/ansys/fluent/core/launcher/launcher.py:541]: @mkundu1 I didn't feel that the current version of the docstring was explicit enough as to the purpose of the argument (it has caused some confusion amongst users) so I had a quick look at the code and surmised what's written above. It's possible that what I've written is not accurate.

File: src/ansys/fluent/core/launcher/launcher.py (hunk 3 above)
Comment (seanpearsonuk) [reply src/ansys/fluent/core/launcher/launcher.py:531]: @mkundu1 it seems that your comment above captures better than either of the docstrings. I will revisit this now.

File: src/ansys/fluent/core/launcher/launcher.py [hunk 4]
```diff
@@ -450,26 +453,26 @@ def launch_fluent(
...
     server instance.
 
     Parameters
     ----------
     product_version : str, optional
-        Version of Fluent to use in the numeric format (such as ``"23.1.0"``
-        for 2023 R1). The default is ``None``, in which case the active version
-        or latest installed version is used.
+        Select an installed version of ANSYS. The string must match the
+        documented version format in the FluentVersion class. The default is
```
Comment (mkundu1) [inline src/ansys/fluent/core/launcher/launcher.py:471]: The version format is not part of docstring in the FluentVersion class. So, if the user is in Jupyter/IDE they will not know the version format unless they look up the doc website.

File: doc/source/api/index.rst [hunk 5]
```diff
@@ -8,25 +8,27 @@ solver components of Fluent.
...
+and part management.
 
 Solver
 ######
 
 The :ref:`ref_solver` mode is dedicated to capturing the power of the Fluent solver. This component consists of a
-:ref:`ref_settings`-based interface or a :ref:`ref_solver_tui`-based interface that is derived from the Fluent
-(solver) TUI, as well as access to Fluent surface, scalar and vector field data.
+:ref:`ref_settings`-based interface and a :ref:`ref_solver_tui`-based interface that is derived from the Fluent
+Solver TUI, as well as access to scalar and vector surface field data.
```
Comment (mkundu1) [inline doc/source/api/index.rst:31]: surface data and scalar and vector field data?

File: src/ansys/fluent/core/launcher/launcher.py (hunk 2 above)
Comment (seanpearsonuk) [reply src/ansys/fluent/core/launcher/launcher.py:547]: @mkundu1 I tried this out - I will update this docstring

**Review by mkundu1 — APPROVED** [review]

**Review by hpohekar — APPROVED** [review]

**Review by prmukherj — APPROVED** [review]

**Pull request merged**